src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const headers = document.querySelectorAll('th');
+const tBody = document.querySelector('tBody');
+const rows = document.querySelectorAll('tBody tr');
```

Comment:
These variables `headers`, `tBody`, and `rows` are declared at the top level and can be accessed from anywhere. This is not a good practice as it can lead to errors when the codebase grows. It's better to encapsulate them within a function.